forms-flow-theme/src/overrideClasses/formioComponentOverride.scss

```diff
@@ -6,3 +6,22 @@
 .builder-sidebar .btn {
     width: 100%;
 }
+
+.formio-form, 
+.formio-builder-form {
+    .form-group {
+      margin-bottom: 1rem;
+    }
+  }
+
+.formio-form {
```

Comment:
no need to duplicate below 16 lines , 10 to 15 lines is enough

Reply:
updated